jbpm-wb-showcase/pom.xml

```diff
@@ -547,6 +547,12 @@
 
     <dependency>
       <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-metadata-backend-elasticsearch</artifactId>
+      <scope>runtime</scope>
```

Comment:
Why is this runtime scope?

Reply:
@mbarkley actually a lot of dependencies in that pom are runtime. I understood that those dependencies were not necessary at compile time, and neither mine. That is why that dep is runtime. Previous Lucene Backend dependency  was also runtime.

Reply:
we moved backend dependencies to runtime scope to shorten the gwt classpath.

Reply:
Okay, makes sense to me.